TypeScript/README.md

```diff
@@ -119,16 +119,16 @@ var foo = fn();
 
 	// Also bad
 	function performOperation (): {
-		str: string,
-		num: number
+		str: string;
```

Comment:
Actually, this is correct b/c it's an interface being set as the return type. Apparently confusion like ours is one of the reasons this is an example of how not to do it ;)